src/scripts/main/toast/index.js

```diff
@@ -35,7 +36,7 @@ class Toast {
             type: "",
             icon: "",
             dismissable: true,
-            displayLength: 4000,
+            displayLength: 400000,
```

Comment:
Hmm 🤔 I guess this remains from testing, or? :)

Reply:
Yup, haha, forgot about that